test/sagemaker_tests/huggingface_tensorflow/training/integration/sagemaker/test_smdataparallel.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2017-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Are there no HF TensorFlow SMD-MP tests?

Reply:
Not yet. HF is in the process of changing TF trainer which will be used for SM integration. We will work on tests/examples after that is ready